examples/energy-management-app/energy-management-common/energy-management-app.zap

```diff
@@ -1452,6 +1452,22 @@
               "maxInterval": 65534,
               "reportableChange": 0
             },
+            {
+              "name": "SupportedWiFiBands",
```

Comment:
this will be problematic for thread platform running this app with common zap file

Reply:
It's not the case right now, and without a good sample that can actually run on Embedded Wi-Fi (which all-clusters is not), I have no way to test.